MAUI/TreeMap/accessibility.md

```diff
@@ -13,9 +13,7 @@ The TreeMap is designed to effectively work with its elements, providing voice d
 
 ## Keyboard shortcuts in .NET MAUI TreeMap (SfTreeMap)
 
-The `Syncfusion .NET MAUI TreeMap` supports keyboard shortcuts for user interaction. 
```

Comment:
As we have already added API links in tree map UG content and merged changes in hotfix branch. So, if you want to ensure images in help staging - first give PR with API links changes from hotfix or master branch and then add images.